tests/features/user/register.feature

```diff
@@ -0,0 +1,40 @@
+@api
+Feature: User registration
+  As a user I must be able to register to the site and complete my user profile.
+
+  Scenario: User can find the register page
+    Given I am an anonymous user
+    When I am on the homepage
+    And I click "Log in"
+    Then I am at "/user/login"
+
+    When I click "Create new account"
+    Then I am at "/user/register"
+
+
+  Scenario: User can register with minimal required fields
+    Given the following user registration at "/user/register":
+      | Email address      | Test-User@Example.com |
+      | Username           | TeStUSer              |
+      | First name         | Test                  |
+      | Family name        | User                  |
+    Then I should see the success message "A welcome message with further instructions has been sent to your email address."
+
+  Scenario: A moderator can register a user
+    Given users:
+      | name            | roles          |
+      | Mr. Moderator   | Moderator     |
+    And I am logged in as "Mr. Moderator"
+    When I am on the homepage
+    Then I click "People"
```

Comment:
There should be a 'Then I click "Manage"' first because when the login occurs for the moderator, the People link is hidden by JS. The profile links are there at first. The test passes because the link exists in the page but it is hidden but this will fail in UAT probably.